keyvi/include/keyvi/dictionary/dictionary_merger.h

```diff
@@ -220,21 +245,37 @@ class DictionaryMerger final {
 
     TRACE("finished iterating, do final compile.");
 
-    generator->CloseFeeding();
+    generator_->CloseFeeding();
+  }
 
-    generator->SetManifestFromString(manifest_);
-    generator->WriteToFile(filename);
+  void Write(std::ostream& stream) {
+    if (!generator_) {
+      throw merger_exception("not merged yet");
+    }
+
+    generator_->Write(stream);
   }
 
+  template <typename StringType>
```

Comment:
Do you a scenario that this will be different from `std::string` ? 
It fells like that we don't need the `template` here.

Reply:
good point, that allows usage of "const char*" but there is no need for such super-flexibility. Let's stick to std::string.

FWIW I will change it in the other places as well.

Reply:
`const char*` anyways will be converted to `std::string` by additional constructor call, which I think is affordable.